### PR TITLE
Fix some tests

### PR DESCRIPTION
**Pull request by Alwahsh**
I've fixed some of the tests to fix issue #209 partially.

### CODE REVIEW

File: app/models/project.rb [hunk 1]
```diff
@@ -64,8 +64,6 @@ def satellitedir
 
   # Push the existing contents of the satellite repo to the bare repo
   def pushtobare
-    barerepo = Rugged::Repository.new self.barerepopath
-    satelliterepo = Rugged::Repository.new self.satelliterepopath
```
Comment (Alwahsh) [inline app/models/project.rb]: I believe those 2 lines have no use, there are functions defined called barerepo and satelliterepo that return the same thing

File: app/models/project.rb [hunk 2]
```diff
@@ -97,7 +95,7 @@ def init
         Rugged::Repository.init_at self.barerepopath, :bare
         Rugged::Repository.clone_at parent.satelliterepopath, self.satelliterepopath
       end
-      self.pushtobare
+      self.pushtobare unless satelliterepo.empty?
```
Comment (Alwahsh) [inline app/models/project.rb]: This fixes 3 of the tests, we don't want to push something to a bare when it's already empty.

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -2,7 +2,6 @@
 ENV["RAILS_ENV"] ||= 'test'
 require File.expand_path("../../config/environment", __FILE__)
 require 'rspec/rails'
-require 'rspec/autorun'
```
Comment (Alwahsh) [inline spec/spec_helper.rb]: This is depreciated in the new version of rspec

**Comment by rohitpaulk** [comment]
Great work! Travis isn't green yet, but I'm merging this - it's certainly better than what we currently have on master.

**Comment by Alwahsh** [comment]
The other failing tests are probably related to issue #181. I'm trying to investigate about it

**Pull request merged**